iaso/api/payments/views.py

```diff
@@ -1,66 +1,327 @@
+        queryset = PaymentLot.objects.all()
+
+        change_requests_prefetch = Prefetch(
+            "payments__change_requests",
+            queryset=OrgUnitChangeRequest.objects.all(),
+            to_attr="prefetched_change_requests",
+        )
+        queryset = queryset.prefetch_related("payments", change_requests_prefetch)
+
+        change_requests_count = (
+            OrgUnitChangeRequest.objects.filter(payment__payment_lot=OuterRef("pk"))
+            .order_by()
+            .distinct()
+            .values("payment__payment_lot")
+            .annotate(total=Count("id", distinct=True))
+            .values("total")
+        )
+
+        payments_count = (
+            Payment.objects.filter(payment_lot=OuterRef("pk"))
+            .order_by()
+            .distinct()
+            .values("payment_lot")
+            .annotate(total=Count("id", distinct=True))
+            .values("total")
+        )
+
+        queryset = queryset.annotate(
+            change_requests_count=Coalesce(Subquery(change_requests_count, output_field=models.IntegerField()), 0),
+            payments_count=Coalesce(Subquery(payments_count, output_field=models.IntegerField()), 0),
+        )
+        queryset = queryset.filter(created_by__iaso_profile__account=self.request.user.iaso_profile.account).distinct()
+
+        return queryset
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            openapi.Parameter(
+                name="user",
+                in_=openapi.IN_QUERY,
+                description="A comma-separated list of User IDs associated with the payment lots creation",
+                type=openapi.TYPE_STRING,
+            ),
+            openapi.Parameter(
+                name="status",
+                in_=openapi.IN_QUERY,
+                description="A comma-separated list of the possible payment lot status",
+                type=openapi.TYPE_STRING,
+            ),
+            openapi.Parameter(
+                name="parent_id",
+                in_=openapi.IN_QUERY,
+                description="The ID of the parent organization unit linked to the change requests. This should also include child units.",
+                type=openapi.TYPE_INTEGER,
+            ),
+            openapi.Parameter(
+                name="created_at_after",
+                in_=openapi.IN_QUERY,
+                description="The start date for when the lots has been created. Format: YYYY-MM-DD",
+                type=openapi.TYPE_STRING,
+                format=openapi.FORMAT_DATE,
+            ),
+            openapi.Parameter(
+                name="created_at_before",
+                in_=openapi.IN_QUERY,
+                description="The end date for when the lots has been created. Format: YYYY-MM-DD",
+                type=openapi.TYPE_STRING,
+                format=openapi.FORMAT_DATE,
+            ),
+        ]
+    )
+    def list(self, request):
+        orders = request.GET.get("order", "updated_at").split(",")
```

Comment:
Same remark as previous PR.
Is this compatible with `filters.OrderingFilter` ??